Bottom: 1f14caa3fac70696237acb1d19841b36e4359046
Top:    b380738e3fe466cca4d99d7c5f8f9db0f54d267e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-24 13:38:28 +0000

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 81c04f9..2af1920 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1756,7 +1756,7 @@ ghid_draw_everything (BoxType *drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.8
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
